VS 2019 build fails

I attempted build using quite fresh VS 2019 (16.1.0) and http://avaloniaui.net/contributing/build stesp. Build failed, output below.

1>------ Rebuild All started: Project: Avalonia.Build.Tasks, Configuration: Debug Any CPU ------
2>------ Rebuild All started: Project: Avalonia.Remote.Protocol, Configuration: Debug Any CPU ------
1>C:\Program Files\dotnet\sdk\2.2.300-preview-010046\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(143,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview
1>Avalonia.Build.Tasks -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Build.Tasks\bin\Debug\netstandard2.0\Avalonia.Build.Tasks.dll
3>------ Rebuild All started: Project: Avalonia.Base, Configuration: Debug Any CPU ------
2>C:\Program Files\dotnet\sdk\2.2.300-preview-010046\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(143,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview
2>Avalonia.Remote.Protocol -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Remote.Protocol\bin\Debug\netstandard2.0\Avalonia.Remote.Protocol.dll
3>Utilities\WeakEventHandlerManager.cs(166,86,166,99): warning CS0252: Possible unintended reference comparison; to get a value comparison, cast the left hand side to type 'MulticastDelegate'
3>Avalonia.Base -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Base\bin\Debug\netstandard2.0\Avalonia.Base.dll
3>Done building project "Avalonia.Base.csproj".
4>------ Rebuild All started: Project: Avalonia.Animation, Configuration: Debug Any CPU ------
4>Avalonia.Animation -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Animation\bin\Debug\netstandard2.0\Avalonia.Animation.dll
5>------ Rebuild All started: Project: Avalonia.Styling, Configuration: Debug Any CPU ------
5>Avalonia.Styling -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Styling\bin\Debug\netstandard2.0\Avalonia.Styling.dll
6>------ Rebuild All started: Project: Avalonia.Visuals, Configuration: Debug Any CPU ------
6>Avalonia.Visuals -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Visuals\bin\Debug\netstandard2.0\Avalonia.Visuals.dll
7>------ Rebuild All started: Project: Avalonia.Layout, Configuration: Debug Any CPU ------
8>------ Rebuild All started: Project: Avalonia.Markup, Configuration: Debug Any CPU ------
9>------ Rebuild All started: Project: Avalonia.OpenGL, Configuration: Debug Any CPU ------
9>EglInterface.cs(54,34,54,48): warning CS0108: 'EglInterface.GetProcAddress' hides inherited member 'GlInterfaceBase.GetProcAddress(string)'. Use the new keyword if hiding was intended.
9>Avalonia.OpenGL -> C:\Temp\Avalonia\Avalonia\src\Avalonia.OpenGL\bin\Debug\netstandard2.0\Avalonia.OpenGL.dll
9>Done building project "Avalonia.OpenGL.csproj".
7>C:\Program Files\dotnet\sdk\2.2.300-preview-010046\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(143,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview
8>Avalonia.Markup -> C:\Temp\Avalonia\Avalonia\src\Markup\Avalonia.Markup\bin\Debug\netstandard2.0\Avalonia.Markup.dll
7>Avalonia.Layout -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Layout\bin\Debug\netstandard2.0\Avalonia.Layout.dll
10>------ Rebuild All started: Project: Avalonia.Interactivity, Configuration: Debug Any CPU ------
10>Avalonia.Interactivity -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Interactivity\bin\Debug\netstandard2.0\Avalonia.Interactivity.dll
11>------ Rebuild All started: Project: Avalonia.Input, Configuration: Debug Any CPU ------
11>Avalonia.Input -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Input\bin\Debug\netstandard2.0\Avalonia.Input.dll
12>------ Rebuild All started: Project: Avalonia.Controls, Configuration: Debug Any CPU ------
12>Avalonia.Controls -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Controls\bin\Debug\netstandard2.0\Avalonia.Controls.dll
13>------ Rebuild All started: Project: Avalonia.Markup.Xaml, Configuration: Debug Any CPU ------
14>------ Rebuild All started: Project: Avalonia.Logging.Serilog, Configuration: Debug Any CPU ------
15>------ Rebuild All started: Project: Avalonia.Controls.DataGrid, Configuration: Debug Any CPU ------
15>Collections\DataGridGroupDescription.cs(61,33,61,48): warning CS0649: Field 'DataGridPathGroupDescription._valueConverter' is never assigned to, and will always have its default value null
15>DataGrid.cs(101,41,101,64): warning CS0169: The field 'DataGrid._currentCellCoordinates' is never used
15>DataGrid.cs(75,33,75,50): warning CS0169: The field 'DataGrid._validationErrors' is never used
15>DataGrid.cs(140,34,140,57): warning CS0169: The field 'DataGrid._currentCellEditBinding' is never used
15>Utils\CellEditBinding.cs(60,26,60,45): warning CS0414: The field 'CellEditBinding.SubjectWrapper._settingSourceValue' is assigned but its value is never used
15>Collections\DataGridCollectionView.cs(930,65,930,80): warning CS1574: XML comment has cref attribute 'SortDescription' that could not be resolved
15>DataGridColumn.cs(671,117,671,124): warning CS1573: Parameter 'binding' has no matching param tag in the XML comment for 'DataGridColumn.GenerateEditingElement(DataGridCell, object, out ICellEditBinding)' (but other parameters do)
15>Avalonia.Controls.DataGrid -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Controls.DataGrid\bin\Debug\netstandard2.0\Avalonia.Controls.DataGrid.dll
15>Done building project "Avalonia.Controls.DataGrid.csproj".
14>Avalonia.Logging.Serilog -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Logging.Serilog\bin\Debug\netstandard2.0\Avalonia.Logging.Serilog.dll
13>PortableXaml\AvaloniaXamlType.cs(229,29,232,51): warning CS0618: 'SetterValueTypeConverter.ConvertSetterValue(ITypeDescriptorContext, XamlSchemaContext, CultureInfo, Setter, object)' is obsolete: 'TODO: try assosiate Setter.Value property with SetterValueTypeConverter, so far coouldn't make it :('
13>PortableXaml\AvaloniaXamlSchemaContext.cs(118,24,118,73): warning CS0618: 'AvaloniaXamlSchemaContext.ResolveSimpleTypeName(string, string)' is obsolete: 'TODO: remove once it's fixed in Portable.xaml'
13>Converters\SetterValueTypeConverter.cs(26,20,26,98): warning CS0618: 'SetterValueTypeConverter.ConvertSetterValue(ITypeDescriptorContext, XamlSchemaContext, CultureInfo, Setter, object)' is obsolete: 'TODO: try assosiate Setter.Value property with SetterValueTypeConverter, so far coouldn't make it :('
13>Avalonia.Markup.Xaml -> C:\Temp\Avalonia\Avalonia\src\Markup\Avalonia.Markup.Xaml\bin\Debug\netstandard2.0\Avalonia.Markup.Xaml.dll
13>Done building project "Avalonia.Markup.Xaml.csproj".
16>------ Rebuild All started: Project: Avalonia.Themes.Default, Configuration: Debug Any CPU ------
16>Avalonia.Themes.Default -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Themes.Default\bin\Debug\netstandard2.0\Avalonia.Themes.Default.dll
17>------ Rebuild All started: Project: Avalonia.Diagnostics, Configuration: Debug Any CPU ------
18>------ Rebuild All started: Project: Avalonia.DesignerSupport, Configuration: Debug Any CPU ------
18>Avalonia.DesignerSupport -> C:\Temp\Avalonia\Avalonia\src\Avalonia.DesignerSupport\bin\Debug\netstandard2.0\Avalonia.DesignerSupport.dll
17>Avalonia.Diagnostics -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Diagnostics\bin\Debug\netstandard2.0\Avalonia.Diagnostics.dll
19>------ Rebuild All started: Project: Avalonia, Configuration: Debug Any CPU ------
19>Avalonia -> C:\Temp\Avalonia\Avalonia\packages\Avalonia\bin\Debug\netstandard2.0\Avalonia.dll
19>CSC : error CS0006: Metadata file 'C:\Temp\Avalonia\Avalonia\src\Avalonia.DesktopRuntime\bin\Debug\netcoreapp2.0\Avalonia.DesktopRuntime.dll' could not be found
19>Done building project "Avalonia.csproj" -- FAILED.
19>CSC : error CS0006: Metadata file 'C:\Temp\Avalonia\Avalonia\src\Avalonia.DesktopRuntime\bin\Debug\net461\Avalonia.DesktopRuntime.dll' could not be found
19>Done building project "Avalonia.csproj" -- FAILED.
20>------ Rebuild All started: Project: Avalonia.Skia, Configuration: Debug Any CPU ------
21>------ Rebuild All started: Project: Avalonia.ReactiveUI, Configuration: Debug Any CPU ------
22>------ Rebuild All started: Project: Avalonia.Win32, Configuration: Debug Any CPU ------
23>------ Rebuild All started: Project: Avalonia.Direct2D1, Configuration: Debug Any CPU ------
21>Avalonia.ReactiveUI -> C:\Temp\Avalonia\Avalonia\src\Avalonia.ReactiveUI\bin\Debug\netstandard2.0\Avalonia.ReactiveUI.dll
24>------ Rebuild All started: Project: Avalonia.Native, Configuration: Debug Any CPU ------
20>C:\Program Files\dotnet\sdk\2.2.300-preview-010046\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(143,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview
23>Avalonia.Direct2D1 -> C:\Temp\Avalonia\Avalonia\src\Windows\Avalonia.Direct2D1\bin\Debug\netstandard2.0\Avalonia.Direct2D1.dll
25>------ Rebuild All started: Project: ControlCatalog, Configuration: Debug Any CPU ------
22>Avalonia.Win32 -> C:\Temp\Avalonia\Avalonia\src\Windows\Avalonia.Win32\bin\Debug\netstandard2.0\Avalonia.Win32.dll
20>Avalonia.Skia -> C:\Temp\Avalonia\Avalonia\src\Skia\Avalonia.Skia\bin\Debug\netstandard2.0\Avalonia.Skia.dll
26>------ Rebuild All started: Project: Avalonia.X11, Configuration: Debug Any CPU ------
26>Avalonia.X11 -> C:\Temp\Avalonia\Avalonia\src\Avalonia.X11\bin\Debug\netstandard2.0\Avalonia.X11.dll
24>CSC : warning CS2002: Source file 'Generated\LocalInterop.cs' specified multiple times
24>CSC : warning CS2002: Source file 'Generated\Enumerations.cs' specified multiple times
24>CSC : warning CS2002: Source file 'Generated\Structures.cs' specified multiple times
24>CSC : warning CS2002: Source file 'Generated\Interfaces.cs' specified multiple times
24>CSC : warning CS2002: Source file 'Generated\Functions.cs' specified multiple times
24>Avalonia.Native -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Native\bin\Debug\netstandard2.0\Avalonia.Native.dll
24>Done building project "Avalonia.Native.csproj".
27>------ Rebuild All started: Project: Avalonia.Desktop, Configuration: Debug Any CPU ------
25>ControlCatalog -> C:\Temp\Avalonia\Avalonia\samples\ControlCatalog\bin\Debug\netstandard2.0\ControlCatalog.dll
27>Avalonia.Desktop -> C:\Temp\Avalonia\Avalonia\src\Avalonia.Desktop\bin\Debug\netstandard2.0\Avalonia.Desktop.dll
28>------ Rebuild All started: Project: ControlCatalog.Desktop, Configuration: Debug Any CPU ------
28>CSC : error CS0006: Metadata file 'C:\Temp\Avalonia\Avalonia\src\Avalonia.DesktopRuntime\bin\Debug\net461\Avalonia.DesktopRuntime.dll' could not be found
28>CSC : error CS0006: Metadata file 'C:\Temp\Avalonia\Avalonia\packages\Avalonia\bin\Debug\net461\Avalonia.dll' could not be found
28>Done building project "ControlCatalog.Desktop.csproj" -- FAILED.
========== Rebuild All: 26 succeeded, 2 failed, 0 skipped ==========

Thank you @Gillibald, I did as advised. Build went better but still fails. Errors encountered:

*  `error XA0115: Invalid value 'armeabi' in $(AndroidSupportedAbis). This ABI is no longer supported. Please update your project properties.` (twice)
*  `error MSB3030: Could not copy the file "C:\Users\thomas\.nuget\packages\SkiaSharp\1.68.0\build\net45\SkiaSharp.dll.config" because it was not found.` Indeed, no .config files in SkiaSharp 1.68 nuget package.
If the latest .Net Core 2.2.203 is required I suggest updating the build guide. Full output attached.

[build output.txt](https://github.com/AvaloniaUI/Avalonia/files/3088584/build.output.txt)

--- a/samples/ControlCatalog.Android/ControlCatalog.Android.csproj
+++ b/samples/ControlCatalog.Android/ControlCatalog.Android.csproj
@@ -32,7 +32,7 @@
     <EmbedAssembliesIntoApk>True</EmbedAssembliesIntoApk>
     <BundleAssemblies>False</BundleAssemblies>
     <AndroidCreatePackagePerAbi>False</AndroidCreatePackagePerAbi>
-    <AndroidSupportedAbis>armeabi;armeabi-v7a;x86</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
     <Debugger>Xamarin</Debugger>
     <AndroidEnableMultiDex>False</AndroidEnableMultiDex>
     <AotAssemblies>False</AotAssemblies>
@@ -51,7 +51,7 @@
     <EmbedAssembliesIntoApk>True</EmbedAssembliesIntoApk>
     <BundleAssemblies>False</BundleAssemblies>
     <AndroidCreatePackagePerAbi>False</AndroidCreatePackagePerAbi>
-    <AndroidSupportedAbis>armeabi,armeabi-v7a,x86</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a,x86</AndroidSupportedAbis>
     <Debugger>Xamarin</Debugger>
     <AotAssemblies>False</AotAssemblies>
     <EnableLLVM>False</EnableLLVM>

--- a/samples/interop/WindowsInteropTest/Program.cs
+++ b/samples/interop/WindowsInteropTest/Program.cs
@@ -15,7 +15,7 @@ namespace WindowsInteropTest
         {
             System.Windows.Forms.Application.EnableVisualStyles();
             System.Windows.Forms.Application.SetCompatibleTextRenderingDefault(false);
-            AppBuilder.Configure<App>().UseWin32().UseDirect2D1().SetupWithoutStarting();
+            AppBuilder.Configure<App>().UseWin32().UseDirect2D1().UseDataGrid().SetupWithoutStarting();
             System.Windows.Forms.Application.Run(new SelectorForm());
         }
     }

--- a/samples/interop/WindowsInteropTest/WindowsInteropTest.csproj
+++ b/samples/interop/WindowsInteropTest/WindowsInteropTest.csproj
@@ -104,6 +104,10 @@
       <Project>{b09b78d8-9b26-48b0-9149-d64a2f120f3f}</Project>
       <Name>Avalonia.Base</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\src\Avalonia.Controls.DataGrid\Avalonia.Controls.DataGrid.csproj">
+      <Project>{3278f3a9-9509-4a3f-a15b-bdc8b5bff632}</Project>
+      <Name>Avalonia.Controls.DataGrid</Name>
+    </ProjectReference>
     <ProjectReference Include="..\..\..\src\Avalonia.Controls\Avalonia.Controls.csproj">
       <Project>{d2221c82-4a25-4583-9b43-d791e3f6820c}</Project>
       <Name>Avalonia.Controls</Name>

--- a/samples/interop/WindowsInteropTest/WindowsInteropTest.csproj
+++ b/samples/interop/WindowsInteropTest/WindowsInteropTest.csproj
@@ -185,6 +185,5 @@
   </ItemGroup>
   <Import Project="..\..\..\build\Rx.props" />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="..\..\..\build\SkiaSharp.props" />
   <Import Project="..\..\..\build\LegacyProject.targets" />
 </Project>

--- a/src/Android/Avalonia.AndroidTestApplication/Avalonia.AndroidTestApplication.csproj
+++ b/src/Android/Avalonia.AndroidTestApplication/Avalonia.AndroidTestApplication.csproj
@@ -32,7 +32,7 @@
     <EmbedAssembliesIntoApk>True</EmbedAssembliesIntoApk>
     <BundleAssemblies>False</BundleAssemblies>
     <AndroidCreatePackagePerAbi>False</AndroidCreatePackagePerAbi>
-    <AndroidSupportedAbis>armeabi;armeabi-v7a;x86</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
     <Debugger>Xamarin</Debugger>
     <AndroidEnableMultiDex>False</AndroidEnableMultiDex>
     <DevInstrumentationEnabled>True</DevInstrumentationEnabled>
@@ -52,7 +52,7 @@
     <EmbedAssembliesIntoApk>True</EmbedAssembliesIntoApk>
     <BundleAssemblies>False</BundleAssemblies>
     <AndroidCreatePackagePerAbi>False</AndroidCreatePackagePerAbi>
-    <AndroidSupportedAbis>armeabi,armeabi-v7a,x86</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a,x86</AndroidSupportedAbis>
     <Debugger>Xamarin</Debugger>
     <AotAssemblies>False</AotAssemblies>
     <EnableLLVM>False</EnableLLVM>

